### ISSUE TITLE
Autoplay Feature request

### ISSUE THREAD

**Issue by Jinraider** [ZhenjianYang/SoraVoice#1]
Hello.

I know you mentioned that you had to take out some features of the voice mod to ensure future compatibility, but I feel, out of all the features that were included prior to Lite, Autoplay is one of the most important for QOL, especially for dialogue heavy games like TiTS. 

This is especially true since these games only use japanese language, so english speakers can't actually tell when one line of dialogue ends, compared to if the clip just has a brief pause in it.

I know that you probably took it out for good reason, but if you think to reimplement any feature, I'd say autoplay is the most important.

**Comment by Moa4sh** [comment]
I agree, the autoplay is an absolute must for QOL. Especially as someone with tendonitis who doesn't want to be clicking the mouse/pressing the controller constantly. If @ZhenjianYang could implement that on the latest version it would be excellent. 

@Jinraider - What I ended up doing was using the original [SoraVoice](https://github.com/ZhenjianYang/SoraVoice-Deprecated) mod and if you use Steam to play the game you can go to your Steam Library, right-click on the game and choose Properties, go to the Betas tab, and select the latest "beta" version which currently is Dec 2019. I think Steam uses this to preserve prior versions of the game before each update. And that Dec 5th version works fine with the original SoraVoice mod, giving you full autoplay functionality.

**Comment by slyguy65** [comment]
third'd
